cps/tasks/metadata_extract.py

```diff
@@ -134,6 +134,9 @@ def _add_download_tasks_to_worker(self, requested_urls):
             self.message = self.media_url_link + f"<br><br>" \
                            f"Number of Videos: {index + 1}/{num_requested_urls}<br>" \
                            f"Total Duration: {datetime.utcfromtimestamp(total_duration).strftime('%H:%M:%S')}"
+            if self.shelf_title:
+                shelf_url = re.sub(r"/meta(?=\?|$)", r"/shelf", self.original_url) + f"/{self.shelf_id}"
```

Comment:
Is this what it's supposed to do?

* Original URL: http://example.com/meta?param=value
* Modified URL: http://example.com/shelf?param=value/12345

Reply:
@deldesir plz post an actual BEFORE/ARTER example if possible, Thanks!

Reply:
> Is this what it's supposed to do?
> 
> * Original URL: http://example.com/meta?param=value
> * Modified URL: http://example.com/shelf?param=value/12345
> 

Here's what it does:
Original URL: http://example.com/meta?param=value
> * Modified URL: http://example.com/shelf/12345